package/main/default/objects/TV_Serie__c/fields/Logo__c.field-meta.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Logo__c</fullName>
+    <externalId>false</externalId>
+    <formula>IMAGE(Logo_url__c , &quot;logo&quot;, 100,100)</formula>
```

Comment:
Logo_url__c is not required. What if it will be null/empty? What the formula will return?

Reply:
Done